### PR TITLE
Fix bug where list has a grammatical error when saying there is 1 item in the list.

### PR DESCRIPTION
**Pull request by zoranabc201**

### CODE REVIEW

File: src/main/java/seedu/duke/Ui.java [hunk 1]
```diff
@@ -233,6 +233,22 @@ public void displayOnePropertyUnitType(Property property, int i) {
         System.out.println(LINE_BREAK);
     }
 
+    public void displayNoOfProperties(int i) {
+        if (i == 1) {
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Ui.java:237]: Perhaps can try to remove magic literal.

File: src/main/java/seedu/duke/Ui.java [hunk 2]
```diff
@@ -233,6 +233,22 @@ public void displayOnePropertyUnitType(Property property, int i) {
...
+        } else {
+            System.out.println("There are " + i + " properties in this list");
+        }
+    }
+
+    public void displayNoOfClients(int i) {
+        if (i == 1) {
+            System.out.println("There is " + i + " client in the list");
+        } else {
+            System.out.println("There are " + i + " clients in this list");
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Ui.java:248]: Would be good to use `ui.showToUser(message)` in the print statement.

File: src/main/java/seedu/duke/command/CommandListClients.java [hunk 3]
```diff
@@ -12,8 +12,7 @@ public void execute(Ui ui, Storage storage, PropertyList propertyList, ClientLis
         for (int i = 0; i < clientList.getCurrentListSize(); i++) {
             ui.displayOneClient(clientList.getClientList().get(i), i + 1);
         }
-        System.out.println("There are " + clientList.getCurrentListSize()
-                + " clients in this list");
+        ui.displayNoOfClients(clientList.getCurrentListSize());
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/command/CommandListClients.java:15]: Good effort to make the print statement more concise.

**Review by wilsonngja — COMMENTED** [review]
Good effort. There are some minor coding standard violation but does not affect the functionality and can be improved in the next iteration. LGTM.

**Review by ngdeqi — APPROVED** [review]
LGTM

**Pull request merged**